python-package/lightgbm/basic.py

```diff
@@ -3516,6 +3530,31 @@ def refit(self, data, label, decay_rate=0.9, **kwargs):
         decay_rate : float, optional (default=0.9)
             Decay rate of refit,
             will use ``leaf_output = decay_rate * old_leaf_output + (1.0 - decay_rate) * new_leaf_output`` to refit trees.
+        reference : Dataset or None, optional (default=None)
+            reference for ``data``.
+            If this is Dataset for validation, training data should be used as reference.
```

Comment:
```suggestion
```

Please remove this line, which I think was copied from the `train()` docs. There isn't a concept of "validation" datasets for the `refit()` method.

Reply:
got it